src/App.tsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
```

Comment:
It's extra, bc you already put the initial value to the `useState`
```suggestion
  const [query, setQuery] = useState('');
```